Sources/PostgresNIO/New/Data/Range+PostgresCodable.swift

```diff
@@ -191,6 +191,11 @@ extension PostgresRange: PostgresEncodable & PostgresNonThrowingEncodable where
     }
 }
 
+// explicitly conforming to PostgresDynamicTypeEncodable and PostgresDynamicTypeThrowingEncodable because of:
+// https://github.com/apple/swift/issues/54132
+extension PostgresRange: PostgresDynamicTypeThrowingEncodable & PostgresDynamicTypeEncodable
```

Comment:
I really wish during review of the `PostgresRange` PR I'd thought to suggest following the stdlib convention and naming it `Bound` instead of `B` 😕

Reply:
Well then let's fix it! 😛

Reply:
and this is internal anyway. fixing.

Reply:
Fixed and landed #367, sorry for the merge conflict @marius-se